### PR TITLE
Handling signals when editing in list view

### PR DESCRIPTION
**Pull request by ptbrowne**
This addresses https://github.com/flask-admin/flask-admin/issues/977

### CODE REVIEW

File: examples/peewee/app.py [hunk 1]
```diff
@@ -1,6 +1,8 @@
 from flask import Flask
+from datetime import datetime
 
 import peewee
+from playhouse.signals import pre_save, Model
```
Comment (rochacbruno) [inline examples/peewee/app.py]: what is playhouse?

Comment (ptbrowne) [inline examples/peewee/app.py]: Playhouse is the collection of addons that are installed along peewee.
http://peewee.readthedocs.org/en/latest/peewee/playhouse.html

Le mer. 12 août 2015 à 14:48, Bruno Rocha <notifications@github.com> a
écrit :

File: flask_admin/model/base.py [hunk 2]
```diff
@@ -1841,7 +1844,34 @@ def ajax_update(self):
         if self.validate_form(form):
             if self.update_model(form, record):
                 # Success
```
Comment (pawl) [inline flask_admin/model/base.py]: Need to remove this unnecessary comment.

**Comment by ptbrowne** [comment]
Closed since it's been such a long time

**Pull request closed without merging**